charts/hono/values.yaml

```diff
@@ -1701,6 +1704,9 @@ commandRouterService:
   # The configMap to get additional environment variables from for this command router service.
   envConfigMap:
 
+  # The path to the truststore for clients using this command router service.
```

Comment:
Can you please add some wording that explains under which circumstances this value will be used and for what?